packages/web-client/app/utils/token.ts

```diff
@@ -98,24 +103,33 @@ const _tokenDisplayInfoMap: Record<TokenSymbol, DisplayInfo> = {
     description: '',
     icon: 'dai-token',
   },
+  SPEND: {
+    name: 'Spend',
+    symbol: 'SPEND',
+    description: '',
+    icon: 'spend',
+    sign: '§',
+  },
```

Comment:
Would prefer SPEND config be separate from the tokens. It's not technically a token (I think?), and doesn't come up where the other tokens do (eg. in a list of balances where we need to display different info/visuals for each token, or in a dropdown).